src/renderer/views/wallet/send/SendViewETH.tsx

```diff
@@ -1,5 +1,125 @@
-import React from 'react'
+import React, { useCallback, useMemo } from 'react'
 
-export const SendViewETH: React.FC = (): JSX.Element => {
-  return <h1>Send ETH - coming soon ...</h1>
+import * as RD from '@devexperts/remote-data-ts'
+import { Client as EthereumClient } from '@xchainjs/xchain-ethereum'
+import { Asset, BaseAmount } from '@xchainjs/xchain-util'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+import { useObservableState } from 'observable-hooks'
+
+import { Send } from '../../../components/wallet/txs/send/'
+import { SendFormETH } from '../../../components/wallet/txs/send/'
+import { ZERO_BASE_AMOUNT } from '../../../const'
+import { useEthereumContext } from '../../../contexts/EthereumContext'
+import { sequenceTOption } from '../../../helpers/fpHelpers'
+import { getWalletBalanceByAsset } from '../../../helpers/walletHelper'
+import { GetExplorerTxUrl, WalletBalances } from '../../../services/clients'
+import { AddressValidation } from '../../../services/ethereum/types'
+import { NonEmptyWalletBalances, TxHashRD } from '../../../services/wallet/types'
+import { WalletBalance } from '../../../types/wallet'
+
+type Props = {
+  selectedAsset: Asset
+  walletBalances: O.Option<NonEmptyWalletBalances>
+  getExplorerTxUrl: O.Option<GetExplorerTxUrl>
+  reloadFeesHandler: () => void
+}
+
+export const SendViewETH: React.FC<Props> = (props): JSX.Element => {
+  const {
+    selectedAsset,
+    walletBalances: oWalletBalances,
+    getExplorerTxUrl: oGetExplorerTxUrl = O.none,
+    reloadFeesHandler
+  } = props
+
+  const oSelectedWalletBalance = useMemo(() => getWalletBalanceByAsset(oWalletBalances, O.some(selectedAsset)), [
+    oWalletBalances,
+    selectedAsset
+  ])
+
+  const { client$, fees$, txRD$, resetTx, subscribeTx } = useEthereumContext()
+
+  const txRD = useObservableState<TxHashRD>(txRD$, RD.initial)
+  const fees = useObservableState(fees$, RD.initial)
+  const oClient = useObservableState<O.Option<EthereumClient>>(client$, O.none)
+
+  /**
+   * Address validation provided by EthereumClient
+   */
+  const addressValidation = useMemo(
+    () =>
+      FP.pipe(
+        oClient,
+        O.map((client) => client.validateAddress),
+        O.getOrElse((): AddressValidation => (_: string) => true)
+      ),
+    [oClient]
+  )
+
+  /**
+   * estimate fee provided by EthereumClient
+   */
+  const estimateFee = useCallback(
```

Comment:
As mentioned by @sarawut11 , it's currently not a fee, but gas. It will be fixed in next PR.